api/api.go

```diff
@@ -1001,8 +1001,11 @@ query namespaceOrbs ($namespace: String, $after: String!) {
 					logger.Error(fmt.Sprintf("Corrupt Orb %s %s", edge.Node.Name, v.Version), err)
 					continue NamespaceOrbs
 				}
-				orbs.Orbs = append(orbs.Orbs, o)
+			} else {
+				o.HighestVersion = "Unpublish"
```

Comment:
Nit: "Unpublish" would be clearer as "Not published" or "No published versions".

Reply:
sure